wrappers/java/README.md

```diff
@@ -0,0 +1,12 @@
+# Ethereum Registration Authority Java Wrapper
+
+
+## Summary
+
+This project provides java interface for interacting with ERA contracts.
+
+
+## Development
+
+To re-generate code in src/main/java/tech/pegasys/era/internal see instructions
+at [https://docs.web3j.io/smart_contracts.html#smart-contract-wrappers](https://docs.web3j.io/smart_contracts.html#smart-contract-wrappers).
```

Comment:
IMO, this is too generic to be useful. Hopefully we'll just remove it all when we implement wrapper auto-regeneration with SID-59 https://pegasys1.atlassian.net/browse/SID-59